src/main/java/com/enderio/base/common/travel/TravelSavedData.java

```diff
@@ -39,12 +40,16 @@ public static TravelSavedData getTravelData(Level level) {
         if (level instanceof ServerLevel serverLevel) {
             return serverLevel.getDataStorage().computeIfAbsent(TravelSavedData::new, TravelSavedData::new, "enderio_traveldata");
         } else {
-            return INSTANCE;
+            if (CLIENT_DATA.containsKey(level.dimension())) {
```

Comment:
Since the client can only be in one dimension at a time, it might be better to keep just one TravelSaveData instance and clear it every time the player changes dimensions.

Reply:
This was done due to me thinking that there is a case where an anchor could still be loaded in one dimension while the player is in the other. While then rejoining the level, the anchor would not be readded to the client list. I am however now wondering if this behaviour can happen, as the client might actually have 1 world where blocks exist, while all the others are removed. I will verify this.